frontend/express/public/core/events/stylesheets/_main.scss

```diff
@@ -53,38 +51,97 @@
         font-family: t.$font-family;
         color: #333c48;
     }
-    &--green {
-        color: #12AF51;
-        padding-left: 8px;
-        font-size: 14px;
+    &__trend {
         font-weight: 500;
         line-height: 14px;
     }
-    &--red {
-        padding-left: 8px;
-        color: red;
-        font-size: 14px;
-        font-weight: 500;
-        line-height: 14px;
+    &__arrow {
+        padding-right: 3px;
     }
 }
 .cly-vue-events-overview-top-events {
     padding-left: 24px;
     padding-bottom: 16.5px;
+    &--tooltip {
+        margin-left: 5.5px;
+    }
+    &__description {
+        font-weight: 500;
+        line-height: 14px;
+    }
+    &__trend {
+        padding-right: 3px;
+    }
 }
 .cly-vue-events-overview-monitor-events {
+    &__charts{
+    .cly-vue-chart__echart{
+        padding: 0px;
+    }
+    }
+    &__empty {
+        min-height: 100px;
+        &__block {
+            margin: 32px auto 32px auto;
+            max-width: 586px;
+        }
+        &--image {
+            width: 242px;
+            height: 242px;
+            background-image: url('../../../images/dashboard/events-overview-noevents.svg');
+        }
+    }
+    &--text{
```

Comment:
why dont you use class - **.text-small** ?

Reply:
The values of properties are different for that. Hence using this.

Reply:
hmmmm.
which component is this again?
can you share an image here.

Reply:
![Uploading Screenshot from 2021-08-31 14-05-39.png…]()

Reply:
![time-period](https://user-images.githubusercontent.com/24314297/131490632-c88b6754-a259-468c-8050-3cf640308522.png)

Reply:
you could have used classes - .text-button .font-weight-bold.
check this and fix this in the next pr.
if it works remove this class.